### PR TITLE
Restaurant page code review.

### PR DESCRIPTION
**Pull request by dipbazz**
In this pull request, the features added are:

- User can view the homepage.
- User can navigate to the menu link and view a list of all menus item.
- User can navigate to the about and contact section.

For every navigation link, I have created a module with specific HTML content to it. Different module present are:

- `pageLoad.js` contains the header with nav links for the site.
- `homePage.js` contains the information displayed on the landing page for home.
- `menuPage.js` contains the list of menus on a card with information about the food item and prices.
- `aboutPage.js` contains the information about the project.
- `contactPage.js` contains the contact form.
- `index.js` loads all the different modules and also contains the logic for the navigation menu link to change on click.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    "eslint": "^6.8.0",
+    "eslint-config-airbnb-base": "^14.1.0",
+    "eslint-plugin-import": "^2.20.2",
+    "stylelint": "^13.3.3",
+    "stylelint-config-standard": "^20.0.0",
+    "stylelint-csstree-validator": "^1.9.0",
+    "stylelint-scss": "^3.17.2",
+    "webpack": "^5.24.4",
+    "webpack-cli": "^4.5.0"
+  },
```
Comment (addod19) [inline package.json]: Kindly add the following to the devDependencies ` webpack, webpack-cli, webpack-dev-server`

File: package.json [hunk 2]
```diff
@@ -0,0 +1,34 @@
+{
+  "name": "restaurant_page",
+  "version": "1.0.0",
+  "description": "Restaurant page using JavaScript where user can view menu's and get the contact details.",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "build": "webpack",
+    "watch": "webpack --watch"
+  },
```
Comment (addod19) [inline package.json]: Let us improve our script so we do away with the warnings when we run the command `npm run build/watch`. Use this instead 
```json
"scripts": {
    "dev": "webpack --mode development",
    "build": "webpack --mode production",
    "start": "webpack-dev-server --mode development --open" // optional
  },
```

**Review by addod19 — CHANGES_REQUESTED** [review]
## Status: Require changes :recycle: :traffic_light: 

Hi @dipbazz, congratulations on this project :muscle: Awesome job :+1: The following requirements have been met:
- README is professional :100: 
- PR is detailed with information :1st_place_medal: 
- Commit messages are easy to understand even for non-techincal guys
- Application works as expected :+1: 
- Great use of ES6 features :smile: 

However, we need to fix the following comments so we get our project approved :+1: Kindly check the comments below:
- Kindly install the following devDependencies; webpack, webpack-cli, webpack-dev-server
- Also let us improve our script so we can run the npm commands without any warnings

Once these are fixed, kindly submit another code review request :slightly_smiling_face: 
_Happy coding_ :man_technologist: 
[Daniel Larbi Addo](https://www.linkedin.com/in/daniel-larbi-addo/)

File: dist/style.css [hunk 3]
```diff
@@ -0,0 +1,219 @@
...
+}
+
+.nav-menu-link::before {
+  content: "";
+  width: 20%;
+  height: 3px;
+  position: absolute;
+  bottom: 0;
+  /* left: 50%;
+  transform: translateX(-50%); */
```
Comment (teekaytech) [inline dist/style.css]: Kindly remove all commented blocks/lines of code.

**Review by teekaytech — APPROVED** [review]
## Status: Approved :green_circle: 

### Congratulations @dipbazz :tada: 

**This project has passed all the mandatory requirements. It is therefore approved :heavy_check_mark:**

**Comment(s)**
- Kindly check my inline comment below, do ensure that the commented lines of code are deleted before you merge.

Again, congratulations!
Do not forget to fill the project completion form.

**Good luck with your next task.**

* * * 
**Taofeek Olalere
[Twitter](https://twitter.com/ola_lere) | [Github](https://github.com/teekaytech) | @taofeekolalere on Slack**
__Happy Coding :smiley:__

**Pull request merged**